permissions/component-bridge-method-injector.yml

```diff
@@ -0,0 +1,8 @@
+---
+name: "bridge-method-injector"
+github: "jenkinsci/bridge-method-injector"
+paths:
+  - "com/infradna/tool"
```

Comment:
I believe the path one level up is needed because this is a Maven plugin (see also `component-kohsuke-maven-plugins.yml` and `component-maven-hpi-plugin.yml` where the same pattern is employed).